src/main.rs

```diff
@@ -195,110 +154,23 @@ async fn get_package_count() -> i32 {
 
     if json["package_managers"].is_array() {
         let pm = json["package_managers"].as_array().unwrap();
-        for i in 0..pm.len() {
-            match pm[i].to_string().trim_matches('\"') {
-                "pacman" => {
-                    let reader = { Exec::cmd("pacman").arg("-Q") | Exec::cmd("wc").arg("-l") }
-                        .stream_stdout()
-                        .unwrap();
-                    commands.push(reader);
-                }
-                "apt" => {
-                    let reader = {
-                        Exec::cmd("dpkg-query").arg("-l")
-                            | Exec::cmd("grep").arg("ii")
-                            | Exec::cmd("wc").arg("-l")
-                    }
-                    .stream_stdout()
-                    .unwrap();
-                    commands.push(reader);
-                }
-                "xbps" => {
-                    let reader = { Exec::cmd("xbps-query").arg("-l") | Exec::cmd("wc").arg("-l") }
-                        .stream_stdout()
-                        .unwrap();
-                    commands.push(reader);
-                }
-                "portage" => {
-                    let reader =
-                        { Exec::cmd("eix-installed").arg("-a") | Exec::cmd("wc").arg("-l") }
-                            .stream_stdout()
-                            .unwrap();
-                    commands.push(reader);
-                }
-                "apk" => {
-                    let reader = { Exec::cmd("apk").arg("info") | Exec::cmd("wc").arg("-l") }
-                        .stream_stdout()
-                        .unwrap();
-                    commands.push(reader);
-                }
-                "dnf" => {
-                    let reader = {
-                        Exec::cmd("dnf").args(&["list", "installed"])
-                            | Exec::cmd("tail").args(&["-n", "+2"])
-                            | Exec::cmd("wc").arg("-l")
-                    }
-                    .stream_stdout()
-                    .unwrap();
-                    commands.push(reader);
-                }
-                _ => (),
-            }
+        for arg in pm {
```

Comment:
i'm actually not sure if this is okay or if the `for i in 0..pm.len()` was required here